### DISCUSSION TITLE
Resilio Sync ico

### DISCUSSION THREAD

**Discussion by icf20** [Lissy93/dashy#1377] [Q&A] [answered]
hi, 

1 the icon for the app is on /gui/favicon.ico
2 if the gui is user/password protected the icon cannot load because is inside the gui, duh

is there any way to load the icon from local resorces?

tia

**Comment by jcunix** [comment, marked as answer]
Icf, this was a bit overkill but I personally built an nginx server, deployed icons I downloaded pushed them to my nginx server then used a url to declare the icon under the edit function or under conf.yml

        title: Title 
        icon: https://ipoficon/qnap.png
        url: https://
        target: newtab
        statusCheck: true
        id: 4_1513_anakinqnap
